libs/auth/src/config.ts

```diff
@@ -4,6 +4,46 @@ import { isVxDev } from '@votingworks/utils';
 import { getRequiredEnvVar } from './env_vars';
 import { FileKey, TpmKey } from './keys';
 
+function shouldUseProdCerts(): boolean {
+  return process.env.NODE_ENV === 'production' && !isVxDev();
+}
+
+function getVxCertAuthorityCertPath(): string {
+  return shouldUseProdCerts()
+    ? // We can commit this prod cert to the codebase because it's 1) universal and 2) public
+      path.join(__dirname, '../certs/prod/vx-cert-authority-cert.pem')
+    : path.join(__dirname, '../certs/dev/vx-cert-authority-cert.pem');
+}
+
+function getMachineCertPathAndPrivateKey(): {
+  certPath: string;
+  privateKey: FileKey | TpmKey;
+} {
+  const machineType = getRequiredEnvVar('VX_MACHINE_TYPE');
+  const machineCertFileName =
+    machineType === 'admin'
+      ? 'vx-admin-cert-authority-cert.pem'
+      : `vx-${machineType}-cert.pem`;
+  if (shouldUseProdCerts()) {
+    const configRoot = getRequiredEnvVar('VX_CONFIG_ROOT');
+    return {
+      certPath: path.join(configRoot, machineCertFileName),
+      privateKey: { source: 'tpm' },
+    };
+  }
+  return {
+    certPath: path.join(__dirname, '../certs/dev', machineCertFileName),
+    privateKey: {
+      source: 'file',
+      path: path.join(
+        __dirname,
+        '../certs/dev',
+        `vx-${machineType}-private-key.pem`
+      ),
+    },
+  };
```

Comment:
In subsequent commits, I generate all the new non-VxAdmin machine keys and certs that we need for dev